node-commons/src/main/java/org/apache/iotdb/commons/client/ClientManager.java

```diff
@@ -56,7 +56,9 @@ public V borrowClient(K node) throws IOException {
       throw e;
     } catch (Exception e) {
       // external doesn't care of other exceptions
-      logger.error(String.format("Borrow client from pool for node %s failed.", node), e);
+      String errorMessage = String.format("Borrow client from pool for node %s failed.", node);
+      logger.error(errorMessage, e);
+      throw new IOException(errorMessage, e);
```

Comment:
Can you check all existing places where clients are obtained through clientManager and remove null judgments? As far as I know, ConfigNode is also using clientManager

Reply:
Do you want me to try to remove all the relevant null judgment?

Reply:
Yes